cgi-bin/LJ/Talk.pm

```diff
@@ -1612,25 +1612,17 @@ sub talkform {
 
     # hidden values
     $template_args->{'hidden_form_elements'} .= LJ::html_hidden(
-        "replyto", $opts->{replyto},
-        "parenttalkid", ( $opts->{replyto} + 0 ),
-        "itemid",   $opts->{ditemid},
-        "journal",  $journalu->{'user'},
-        "basepath", $basepath,
-        "dtid",     $opts->{dtid},
-        "editid",   $editid,
-        %{ $opts->{styleopts} },
+        "replyto"      => $opts->{replyto},
+        "parenttalkid" => ( $opts->{replyto} + 0 ),
+        "itemid"       => $opts->{ditemid},
+        "journal"      => $journalu->{'user'},
+        "basepath"     => $basepath,
+        "dtid"         => $opts->{dtid},
+        "editid"       => $editid,
+        "chrp1"        => generate_chrp1($journalu->{userid}, $opts->{ditemid}),
+        %$styleopts,
```

Comment:
(Sorry, ignore that uninitialized local variable; it's a glitch in the timestream from rebasing. That variable gets assigned during the "reply forms maintenance" commit.)

Reply:
- [ ] 👍